satellite/satellitedb/migrate.go

```diff
@@ -504,7 +504,16 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 						created_at timestamp with time zone NOT NULL,
 						PRIMARY KEY ( project_id, limit_type )
 					);`,
-					`ALTER TABLE projects DROP COLUMN usage_limit;`,
+					`WITH limits as (SELECT id, usage_limit FROM projects WHERE usage_limit > 0);
```

Comment:
What happens when new API tries to insert into `project_limits` while this migration step is running?